### PR TITLE
{bio,tools}[intel/2019b] BiG-SCAPE v1.0.1, networkx v2.4 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 5 out of 5 (2 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/b78d479e1903a3d036fb3cd6454a4924 for a full test report.

File: easybuild/easyconfigs/b/BiG-SCAPE/BiG-SCAPE-1.0.1-intel-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+sanity_check_commands = [
+    'bigscape.py --help',
+]
+
+modextrapaths = {'PATH': ''}
+
+modloadmsg = "%(name)s needs processed Pfam database to work properly.\n"
+modloadmsg += "For this, download the latest 'Pfam-A.hmm.gz' file from the Pfam website"
+modloadmsg += "(ftp://ftp.ebi.ac.uk/pub/databases/Pfam/releases/), "
```
Comment (Micket) [inline easybuild/easyconfigs/b/BiG-SCAPE/BiG-SCAPE-1.0.1-intel-2019b-Python-3.7.4.eb:45]: actually, can't we make a easyconfig for the database as well, and depend on it?

Comment (boegel) [reply easybuild/easyconfigs/b/BiG-SCAPE/BiG-SCAPE-1.0.1-intel-2019b-Python-3.7.4.eb:45]: I doubt installing databases where software installed is good practice?

Comment (Micket) [reply easybuild/easyconfigs/b/BiG-SCAPE/BiG-SCAPE-1.0.1-intel-2019b-Python-3.7.4.eb:45]: hm, ok they were a bit larger than expected

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/73ef0a8464561985ae51d9deda25f4be for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/bfe5843f942d5369cca18f9500e30ac5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2611.swalot.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/08088dd91c072e40014048848b76b495 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @Darkless012!

**Pull request merged**